pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting.md

```diff
@@ -0,0 +1,275 @@
+```
+<table>
+  <tr>
+   <td>
+    <code>$title</code>
+   </td>
+   <td>The title of your document as it will appear in the table of contents. Capitalize the first letter of the first word, except for “AMP” and other proper nouns. Use the ampersand `&` instead of the word `and`.
+   </td>
+  </tr>
+  <tr>
+   <td>
+    <code>$order</code>
+   </td>
+   <td>Define where in the table of contents your document appears. You may need to edit the `$order` in other documents for it to appear in the correct position. 
+   </td>
+  </tr>
+  <tr>
+   <td>
+    <code>formats</code>
+   </td>
+   <td>List the AMP experiences your document is relevant to. If your document was relevant to AMP websites and AMP stories, but not AMP ads or AMP email, your frontmatter would like the following:
+    ```yaml
+        formats:
+          - websties 
+          - stories
+    ```  
+   </td>
+  </tr>
+  <tr>
+   <td>
+<code>author</code>
+   </td>
+   <td>The author is you! Use your GitHub username. 
+   </td>
+  </tr>
+  <tr>
+   <td>
+<code>contributors</code>
+   </td>
+   <td>List anyone who contributed to your document. This field is optional.
+   </td>
+  </tr>
+  <tr>
+   <td>
+<code>description</code>
+   </td>
+   <td>Write a brief description of your guide or tutorial. This helps with search engine optimization, getting your work into the hands of those who need it! 
+   </td>
+  </tr>
+  <tr>
+   <td>
+<code>tutorial</code>
+   </td>
+   <td>Add `tutorial: true` to the frontmatter for the website to add the tutorial icon next to it. Tutorials are at the bottom of their section in the table of contents.
+   </td>
+  </tr>
+</table>
+
+# Shortcodes
+
+For a list of shortcodes and their uses, please view [documentation.md on GitHub](https://github.com/ampproject/amp.dev/blob/future/contributing/documentation.md#shortcodes).
+
+## Filtering sections
+Some documents may be relevant for multiple AMP formats, but certain formats may need further explanation or information that is not relevant to the others. You can filter these sections by wrapping them in the following shortcode.
+
+<div class="ap-m-code-snippet">
+<pre>
+&lsqb;filter formats="websites"]
+This is only visible for [websites](?format=websites).
+&lsqb;/filter]
+
+&lsqb;filter formats="websites"]
+This is only visible for [websites](?format=websites).
+&lsqb;/filter]
+
+&lsqb;filter formats="websites, email"]
+This is visible for [websites](?format=websites) & [email](?format=email).
+&lsqb;/filter]
+
+&lsqb;filter formats="stories"]
+This is visible for [stories](?format=stories).
+&lsqb;/filter]
+</pre>
+</div>
+
+## Tips
+You can add tips and callouts by wrapping text in the following shortcode:
+
+<div class="ap-m-code-snippet">
+<pre>
+&lsqb;tip type="default"]
+Default tip
+[/tip]
+
+&lsqb;tip type="important"]
+Important
+[/tip]
+
+&lsqb;tip type="note"]
+Note
+[/tip]
+
+&lsqb;tip type="read-on"]
+Read-on
+[/tip]
+</pre>
+</div>
+
+## Code samples
```

Comment:
This section is an incomplete copy of: https://github.com/ampproject/amp.dev/blob/future/contributing/documentation.md#code-samples. 

We should link to the page and at least mention the executable inline samples here.